### PR TITLE
Remove reference to MD5 in Unrestricted File Upload article

### PR DESCRIPTION
**Pull request by poletela-na-mars**
It's a good idea to write another example of hashing algorithm (Keccak, other ideas) due to the large number of ways to crack MD5

### CODE REVIEW

File: pages/vulnerabilities/Unrestricted_File_Upload.md [hunk 1]
```diff
@@ -369,7 +369,7 @@ And some special recommendations for the developers and webmasters:
     name of a file plus its extension should be less than 255 characters
     (without any directory) in an NTFS partition.
   - It is recommended to use an algorithm to determine the filenames.
-    For instance, a filename can be a MD5 hash of the name of file plus
+    For instance, a filename can be a Keccak hash of the name of file plus
```
Comment (kingthorin) [inline pages/vulnerabilities/Unrestricted_File_Upload.md:372]: Why not SHA3?

Even better why not remove the algo altogether and just leave "a hash" then no future update is necessary.

Comment (poletela-na-mars) [reply pages/vulnerabilities/Unrestricted_File_Upload.md:372]: Actually, that's what I meant (isn't it known as SHA-3?) Yeah, it's better to leave just "a hash"

Comment (kingthorin) [reply pages/vulnerabilities/Unrestricted_File_Upload.md:372]: 😀 you using "keccak" actually made me go read about this. Apparently when NIST adopted Keccak they actually changed the padding slightly so while it's the same algo it uses a different padding and so it's technically not the exact same :shrug: Learn something everyday 😄

Comment (kingthorin) [reply pages/vulnerabilities/Unrestricted_File_Upload.md:372]:
```suggestion
    For instance, a filename can be a hash of the name of file plus
```

**Review by kingthorin — APPROVED** [review]
Thanks

File: pages/vulnerabilities/Unrestricted_File_Upload.md (hunk 1 above)
Comment (poletela-na-mars) [reply pages/vulnerabilities/Unrestricted_File_Upload.md]: Oh, I didn't know, thanks)

**Pull request merged**